shell/src/main/java/alluxio/cli/fs/command/MountCommand.java

```diff
@@ -77,14 +84,15 @@ public String getCommandName() {
   @Override
   public Options getOptions() {
     return new Options().addOption(READONLY_OPTION).addOption(SHARED_OPTION)
-        .addOption(OPTION_OPTION);
+        .addOption(EXCLUDE_UFS_OPTION).addOption(OPTION_OPTION);
   }
 
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
-    if (args.length == 0) {
-      Map<String, MountPointInfo> mountTable = mFileSystem.getMountTable();
+    if (args.length == 0 || cl.hasOption(EXCLUDE_UFS_OPTION.getLongOpt())) {
```

Comment:
What logic is your expected?

I guess what you want is 

```java
if (args.length == 0) {
    Map<String, MountPointInfo> mountTable =
          mFileSystem.getMountTable(!cl.hasOption(EXCLUDE_UFS_OPTION.getLongOpt()));
      UfsCommand.printMountInfo(mountTable);
}
```

Reply:
yes it's better.